samples/productSearch/productSearch.v1p3beta1.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function addProductToProductSet(projectId, location, productId, productSetId) {
+  // [START vision_product_search_add_product_to_product_set]
+
+  const vision = require('@google-cloud/vision').v1p3beta1;
+
+  var client = new vision.ProductSearchClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = 'Your Google Cloud project Id';
+  // const location = 'A compute region name';
+  // const productId = 'Id of the product';
+  // const productSetId = 'Id of the product set';
+
+  var productSetPath = client.productSetPath(projectId, location, productSetId);
+
+  var productPath = client.productPath(projectId, location, productId);
+
+  var request = {
+    name: productSetPath,
+    product: productPath,
+  };
+
+  client
+    .addProductToProductSet(request)
+    .then(() => {
+      console.log(`Product added to product set.`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END vision_product_search_add_product_to_product_set]
+}
+
+function listProductsInProductSet(projectId, location, productSetId) {
+  // [START vision_product_search_list_products_in_product_set]
+
+  const vision = require('@google-cloud/vision').v1p3beta1;
+
+  var client = new vision.ProductSearchClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = 'Your Google Cloud project Id';
+  // const location = 'A compute region name';
+  // const productSetId = 'Id of the product set';
+
+  var productSetPath = client.productSetPath(projectId, location, productSetId);
+
+  var request = {
+    name: productSetPath,
+  };
+
+  //client
```

Comment:
Remove commented out code

Reply:
This has been resolved as part of https://github.com/googleapis/nodejs-vision/pull/196#